Minor stuff and setup the NetBox Request Queue Worker

ypid opened the pull request.

ypid committed:
[netbox] Remove redundant copyright header from template file

ypid committed:
[cryptsetup] Clearly threat of quantum computers to cryptsetup

ypid committed:
[cryptsetup] Review luks_device module -> Currently too limited

ypid committed:
Fix debops-contrib-playbooks after role rename on 2020-08-19

ypid committed:
[netbox] Update to v2.9.4

ypid committed:
[netbox] Setup the NetBox Request Queue Worker so that reports work

ypid committed:
[tor] Fix default variable help texts

ypid committed:
[docs] Update my DebOps philosophy entry

drybjed commented on ansible/roles/netbox/handlers/main.yml:
```diff
@@ -17,3 +17,10 @@
+
+- name: Restart netbox Request Queue Worker
+  service:
+    name: 'netbox'
```
It should probably be named `netbox-rq`.

ypid replied:
Yes, that makes more sense :) Fixed.

ypid committed:
Add myself as copyright holder of CHANGELOG.rst

```Shell
git shortlog -sn -- CHANGELOG.rst | head -n 2
   539  Maciej Delmanowski
    44  Robin Schneider
```

drybjed reviewed (approved):
Nicely done, thanks.
